tests/pipelines/ip_adapters/test_ip_adapter_stable_diffusion.py

```diff
@@ -544,3 +544,33 @@ def test_ip_adapter_multiple_masks(self):
 
         max_diff = numpy_cosine_similarity_distance(image_slice, expected_slice)
         assert max_diff < 5e-4
+
+    def test_ip_adapter_multiple_masks_one_adapter(self):
```

Comment:
This is slow test. I was wondering if it's possible to add a fast test like

https://github.com/huggingface/diffusers/blob/c2e87869be81bef9c87adb34b7815adebe042e64/tests/pipelines/test_pipelines_common.py#L238

?